External/FEXCore/Source/Interface/Core/JIT/Arm64/VectorOps.cpp

```diff
@@ -40,15 +40,37 @@ DEF_OP(VectorImm) {
   auto Op = IROp->C<IR::IROp_VectorImm>();
 
   const uint8_t OpSize = IROp->Size;
-  const uint8_t Elements = OpSize / Op->Header.ElementSize;
+  const uint8_t ElementSize = Op->Header.ElementSize;
+  const uint8_t Elements = OpSize / ElementSize;
 
-  if (Op->Header.ElementSize == 8) {
-    // movi with 64bit element size doesn't do what we want here
-    LoadConstant(TMP1.X(), Op->Immediate);
-    dup(GetDst(Node).V2D(), TMP1.X());
-  }
-  else {
-    movi(GetDst(Node).VCast(OpSize * 8, Elements), Op->Immediate);
+  if (CanUseSVE) {
```

Comment:
I think `CanUseSVE` should be called more like "Support256Vectors"

Reply:
Agreed. `HostSupportsSVE` is way more descriptive. Will migrate that over after the current IR PRs get merged